plugins/progress/example.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source $OSH/plugins/progress/progress2.plugin.sh
+
+for i in {0..101}; do progress $i; done
```

Comment:
typo?
```suggestion
for i in {0..101}; do progress2 $i; done
```

Also, why does the above example finally call `progress2 101`?